diff -up libappimage-1.0.4-5/src/libappimage/utils/hashlib.h.omv~ libappimage-1.0.4-5/src/libappimage/utils/hashlib.h
--- libappimage-1.0.4-5/src/libappimage/utils/hashlib.h.omv~	2023-04-19 02:30:53.438296743 +0200
+++ libappimage-1.0.4-5/src/libappimage/utils/hashlib.h	2023-04-19 02:30:59.611344519 +0200
@@ -2,7 +2,7 @@
 
 // system
 #include <istream>
-
+#include <cstdint>
 
 
 namespace appimage {
